docker/run-tests.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/sh
 
 epmd -daemon
-./rebar3 as all_tests do dialyzer, eunit, ct
+apt-get install -y pcregrep
+mkdir -p /vernemq/_build/all_tests+test/logs
+./rebar3 as all_tests do dialyzer, eunit, ct ||
```

Comment:
If we have an executable shell file, we can simply call that from here, and any changes to it, will reflect in both CI runner and docker runner.

Reply:
Done